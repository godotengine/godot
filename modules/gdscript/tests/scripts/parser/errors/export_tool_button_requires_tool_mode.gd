@export_tool_button("Click me!") var action
