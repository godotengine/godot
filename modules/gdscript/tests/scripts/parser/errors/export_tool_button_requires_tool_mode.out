GDTEST_ANALYZER_ERROR
Tool buttons can only be used in tool scripts (add "@tool" to the top of the script).
